src/makeCalculator.js

```diff
@@ -4,7 +4,30 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const abilities = {
+    result: 0,
+    add: (number) => (abilities.result += number),
+    subtract: (number) => (abilities.result -= number),
+    multiply: (number) => (abilities.result *= number),
+    divide: (number) => (abilities.result /= number),
```

Comment:
[CODE STYLE] - It's good practice to use arithmetic operators with assignment operator `=` when you want to add, subtract, multiply, or divide the existing value of a variable.

You are already doing this correctly, so no changes are required here. Keep in mind that this is the preferred style for such operations.